3-deployment-targets-adapters-and-stacks/bee-rich/README.md

```diff
@@ -26,13 +26,13 @@ Let's get started! Follow the instructions in _Chapter 3, Deployment Targets, Ad
 - `npm run format` - runs all the formatting scripts
 - `npm run format:lint` - runs eslint and fixes errors
 - `npm run clean` - deletes all generated files and folders and installs dependencies
-- `npm run build` - runs all the build scripts
 - `npm run build:css` - generates the CSS for production
 - `npm run build:remix` - builds the Remix application
-- `npm run dev` - runs all the dev scripts in parallel
+- `npm run build` - runs all the build scripts
 - `npm run dev:css` - generates and watches the CSS for development
 - `npm run dev:node` - runs the server in development mode
 - `npm run dev:remix` - runs the Remix watcher
+- `npm run dev` - runs all the dev scripts in parallel
```

Comment:
I rearranged these slightly because I thought it might make more sense to think to put the `build` and `dev` scripts after the specific `build:`/`dev:` scripts. Might make more sense to someone who hasn't seen this pattern 🤷‍♂️